### DISCUSSION TITLE
Welche Firmware für OTA? _esp32s3.bin oder _esp32s3_usb.bin ?

### DISCUSSION THREAD

**Discussion by Manos1966** [tbnobody/OpenDTU#2141] [Q&A]
Die Frage(n), die ich immer stellen wollte 🙈 

Ich habe jetzt ein ESP32-S3-DevKit Board zum testen.
Diese Boards haben zwei USB Ports, ein "Standard USB" sowie ein "USB-to-UART"
Ich verstehe es also so, das Board hat sowohl "usb-uart bridge" als auch "integrated usb connection"
![ESP32-S3-DevKitC-1_v2-annotated-photo USB-to-UART and USB b](https://github.com/user-attachments/assets/59998397-e34b-4fd3-a4e9-d92416842fa3)

Daher meine Frage(n)

1. First Flash (= mit Kabel)
welche Firmware sollte ich für solche ESP32-S3 Boards auswählen?
opendtu-generic_esp32s3.factory.bin  (For boards with usb-uart bridge)
opendtu-generic_esp32s3_usb.factory.bin (For boards with integrated usb connection)
...oder hängt es vom Port ab wo ich das USB Kabel anschließe?

2. OTA Updates
welche Firmware sollte ich  für solche ESP32-S3 Boards auswählen wenn ich ein OTA Update mache?
opendtu-generic_esp32s3.bin  (For boards with usb-uart bridge)
opendtu-generic_esp32s3_usb.bin (For boards with integrated usb connection)

Vielen Dank im voraus!

PS. Ich addiere die Info hier, falls jemand die gleiche Suche machen muss: 
#2064 Für die Fusion DTU Boards muss die folgende Firmware geflasht werden:
opendtu-generic_esp32s3_usb.bin (OTA)
opendtu-generic_esp32s3_usb.factory.bin (First Flash)

**Comment by Manos1966** [comment]
Keine Antwort?
@tbnobody Ich fange an, mir Sorgen zu machen, wenn unterschiedliche Firmware Updates geliefert werden, aber aber kaum jemand weiss wofuer... 🤦‍♂️

**Reply by tbnobody** [reply]
Ist alles in der Doku beschrieben. 
Muss man nur lesen. Z.b. https://www.opendtu.solar/firmware/flash_esp/#download-firmware

**Comment by Manos1966** [comment]
> Ist alles in der Doku beschrieben.
Muss man nur lesen.

@tbnobody Wirklich?
"we agree to disagree"
Vielleicht solltest du meine Post genauer lesen...
